### PR TITLE
An alternative version of data.php which converts metric to imperial

### PR DESCRIPTION
**Pull request by DougieLawson**
Added extra code to convert Celsius to Fahrenheit for temp values, kilometre/hour to miles per hour for wind speed and gusts and millimetres to inches for rainfall.

### CODE REVIEW

**Comment by ghollingworth** [comment]
Can it convert the code to cogs dials and spindles as well?  :)

**Comment by DougieLawson** [comment]
I'll see if it can make tea and answer the door bell for the next pull request.

**Comment by ED6E0F17** [comment]
Not everyone is at "112.2m ASL", so I don`t see the value of that conversion. Can you do pressure in mil of Hg ?

**Comment by DougieLawson** [comment]
Correct. Not everyone is at 112.2m. I am, so it works for me. The installation instructions will tell folks to change that to their measured altitude to correct local pressure readings to mean sea level.

If you want to convert hPa to inches of Hg, then write your own code and do a pull request. 

1 hPa == 0.02953 inHg

**Comment by ED6E0F17** [comment]
Most of your users will use the default settings, and ignore any instructions you might write, so it would be a good idea to use 0 for the default altitude, and correct the pressure to MSL only when another value is given.

I like to see imperial measures alongside the SI values, and the conversion to mmHg is easier than I expected, so I will add that to my TODO list.

**Comment by DougieLawson** [comment]
What part of "change the code yourself and create a pull request" are you having a difficulty with?

File: data.imperial.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+$con=mysqli_connect("localhost","weather","WeatherMySQLPasswd","weather");
+$column_name = $_GET["col"];
+$msl = 0;
+if ($column_name == "MSL_PRESSURE"){
+    $column_name = "AIR_PRESSURE";
+    $msl = 1;
+    $altitude = 112.2; // metres
+}
+if ($column_name == "AMBIENT_TEMPERATURE" or $column_name == "GROUND_TEMPERATURE") {
+    $farenheit = 1;
```
Comment (pelwell) [inline data.imperial.php]: "fahrenheit" is the correct spelling.

File: data.imperial.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+$rows = array();
+
+while($row = mysqli_fetch_array($result)) {
+    $time_unix = floatval($row['UNIX_TIME']) * 1000; //flot needs it in milliseconds
+    if ($msl) {
+         $air_value = floatval($row[$column_name]);
+         $msl_value = $air_value/(pow(1-($altitude/44330.0),5.255));
+         $row[$column_name] = $msl_value;
+    }
+    if ($farenheit) {
```
Comment (DougieLawson) [inline data.imperial.php]: Needs to change here if you insist on correcting my bad German.

**Comment by ED6E0F17** [comment]
Lol. I am new to all this github stuff.

**Pull request merged**